rank/data/tests/works.json

```diff
@@ -433,6 +441,18 @@
                 "after": [
                     "thgzs6pd"
                 ]
+            },
+            {
+                "searchTerms": "Mackie",
+                "before": [
+                    "k8ccmj6t",
+                    "s2tnttyn"
+                ],
+                "after": [
+                    "ae5kymcs"
+                ],
+                "description": "An exact matches in the title should appear before a misspelled match in the contributor and title. Also uses a filter",
```

Comment:
Since the filter is just a boolean yes/no, it doesn't affect the order of results.

If we want to test the filter logic is working, can we add an example of something that _shouldn't_ appear because it's being excluded by the filter?

Reply:
Maybe... IMO this set of tests should only exist to validate relevance, not the rank tool itself, and I'm confident that the filters are being applied when a query is run. I think more and more of our tests will start incorporating filters as time goes on, but I'm hesitant to add tests prematurely which aren't motivated by real user needs!

Maybe we can include an additional `Mackie` example in the `precision` tests to really incorporate the filter feature, ie

<details><summary>Searching for Mackie without a filter</summary>

```
✔ index › works-candidate
✔ queryEnv › candidate
✔ What are you looking for? … Mackie
✔ Any filters? … 
1.      Jog on : how running saved my life / Bella Mackie.
        [2018]
        https://wellcomecollection.org/works/kqczv6ks

2.      Tapping the deep roots of Britain's Blacks / Lindsay Mackie.
        1980
        https://wellcomecollection.org/works/vpuqcn4c

3.      Notes of a case of persistent priapism / by John Mackie.
        1872
        https://wellcomecollection.org/works/vdzp9u3d
```

</details>

<details><summary>Searching for Mackie with a filter</summary>

```
✔ index › works-candidate
✔ queryEnv › candidate
✔ What are you looking for? … Mackie
✔ Any filters? … {"term": {"query.format.id": "h"}}
1.      Mackie, Donald
        HALDANE/5/2/3/52
        26 Dec 1952
        https://wellcomecollection.org/works/k8ccmj6t

2.      Mackie J
        WF/M/I/PR/727b
        c.1984
        https://wellcomecollection.org/works/s2tnttyn

3.      ACL Mackie
        SA/ASH/C/18
        1972-1973
        https://wellcomecollection.org/works/y3ghxj83
```

</details>

So our test should ensure that `k8ccmj6t` is first in the list when an archive filter is applied

Reply:
Mmm, okay. That makes sense. (I'm now reading the Slack thread.)

Let's leave this as-is then.